### ISSUE TITLE
Incorrect numbering of the verses + Indentation

### ISSUE THREAD

**Issue by Doc73** [maieul/ledmac#125]
Dear Maieul,
Another issue, after #124 .
As you can see from this sample

``` tex
% !TeX encoding = UTF-8
% !TEX TS-program = pdflatex

\documentclass{scrbook}
\usepackage[LGRx,T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[italian]{babel}

\usepackage{eledmac,eledpar}

\begin{document}

\setstanzaindents{3,4,3,3,3,3,4,3,3,3,3,4,3,3,3,3}

\begin{pages}
\begin{Leftside}
\beginnumbering

\begin{astanza}
%\ledchapter*{English}
First verse &
Second verse &
Third verse &
Fourth verse &
Fifth verse.&

First verse &
Second verse &
Third verse &
Fourth verse &
Fifth verse.&

First verse &
Second verse &
Third verse &
Fourth verse &
Fifth verse.\&
\end{astanza}

\endnumbering
\end{Leftside}

\begin{Rightside}
\beginnumbering

\begin{astanza}
%\ledchapter*{Italiano}
Primo verso &
Secondo verso &
Terzo verso &
Quarto verso &
Quinto verso.&

Primo verso &
Secondo verso &
Terzo verso &
Quarto verso &
Quinto verso.&

Primo verso &
Secondo verso &
Terzo verso &
Quarto verso &
Quinto verso.\&
\end{astanza}

\endnumbering
\end{Rightside}
\Pages
\end{pages}

\chapter{Title}
\setstanzaindents{5,3,2,2,2,2,3,2,2,2,2,3,2,2,2,2,3,2,2,2,2,3,2,2,2,2,3,2,2,2,2}
%\setstanzaindents{2,2,2,2,2}
%\setcounter{stanzaindentsrepetition}{6}
\beginnumbering
\stanza
Primo verso &
Secondo verso &
Terzo verso &
Quarto verso &
Quinto verso.&

Primo verso &
Secondo verso &
Terzo verso &
Quarto verso &
Quinto verso.&

Primo verso &
Secondo verso &
Terzo verso &
Quarto verso &
Quinto verso.&

Primo verso &
Secondo verso &
Terzo verso Nel mezzo del cammin di nostra vita mi ritrovai per una selva oscura ché la diritta via era smarrita&
Quarto verso &
Quinto verso.&

Primo verso Nel mezzo del cammin di nostra vita mi ritrovai per una selva oscura ché la diritta via era smarrita&
Secondo verso &
Terzo verso &
Quarto verso &
Quinto verso.&

Primo verso &
Secondo verso &
Terzo verso &
Quarto verso &
Quinto verso.\&
\endnumbering

\end{document}
```

there are some problems:
1) the numeration of verses is not stable and changes at every compilation
2) the indentation of long lines is not correct in the case that the long line is the first

Thank you very much again and good continuation of yout holidays!
Doc
